packages/core/src/particle/ParticleGenerator.ts

```diff
@@ -261,8 +265,8 @@ export class ParticleGenerator {
           this._resizeInstanceBuffer(false);
         }
       }
-
       emission._emit(lastPlayTime, this._playTime);
+      this._updateBoundingBox();
```

Comment:
The logic is not strict and clear enough，may cause `_updateBoundingBox` and `set 0` to be executed at the same time！

How about optimizing this way?

```ts
if (this.isAlive) {
      this._updateBoundingBox();
    } else {
      // Reset play time when is not playing and no active particles to avoid potential precision problems in GPU
      const discardTime = Math.min(emission._frameRateTime, Math.floor(this._playTime / duration) * duration);
      this._playTime -= discardTime;
      emission._frameRateTime -= discardTime;

      const bounds = this._renderer._bounds;
      bounds.min.set(0, 0, 0);
      bounds.max.set(0, 0, 0);
    }
```